apm-agent-plugins/apm-profiling-plugin/src/test/java/co/elastic/apm/agent/profiler/CallTreeSpanifyTest.java

```diff
@@ -68,6 +69,7 @@ void tearDown() throws IOException {
     }
 
     @Test
+    @DisabledIfSystemProperty(named = "os.name", matches = "Windows 10")
```

Comment:
Doesn't `@DisabledOnOs(OS.WINDOWS)` work here? It should be applied at method level as well.

Reply:
Yes indeed, for some reason I decided that it does not work only on windows 10.
Replaced with `@DisabledOnOs`